style.css

```diff
@@ -303,3 +327,159 @@ input {
   color: #42526e;
   margin-left: 25px;
 }
+
+/* ---------------Mobile Vesrion Ends Here--------------- */
+
+/* -----------------------Desktop Vesrion Started----------------------- */
+
+@media screen and (min-width: 768px) {
+  #headline-img-desktop {
+    display: block;
+    width: 100%;
+  }
+
+  .navbar {
+    display: block;
+    margin-left: 41%;
+  }
+
+  #navbar-list {
+    list-style-type: none;
+  }
+
+  #navbar-list li {
+    display: inline;
+    padding: 12px;
+  }
+
+  a:link {
+    text-decoration: none;
+  }
+
+  #logo-image,
+  .menu,
+  #headline-img-mobile {
+    display: none;
+  }
+
+  .text {
+    padding-bottom: 10%;
+  }
+
+  #links-list {
+    margin-right: 8%;
+  }
+
+  #action {
+    margin-left: 29%;
+  }
+
+  .second-text {
+    width: 91%;
+    height: auto;
+  }
+
+  .cards-group {
+    display: grid;
+    grid-template-columns: auto auto auto;
+    row-gap: 30px;
+    justify-content: center;
+  }
+
+  #Recent-Works-Text {
+    width: 100%;
+    height: auto;
+    padding-left: 2%;
+  }
+
+  #about-text {
+    padding-right: 345px;
+  }
+
+  .about-me {
+    width: 100%;
+    display: grid;
+    row-gap: 30px;
+    justify-content: center;
+  }
+
+  #get-my-resume-btn {
+    margin-left: 27%;
+  }
+
+  .about-detail {
+    width: 900px;
+    height: 96px;
+  }
+
+  .about1 {
+    display: none;
+  }
+
+  .skills {
+    display: grid;
+    grid-template-columns: auto auto auto;
+    justify-content: center;
+  }
+
+  .form-section {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+  }
+
+  #form-detail {
+    display: grid;
+    grid-template-columns: 50% 50%;
+    margin-top: 5%;
+    padding-left: 7%;
+  }
+
+  .form-description {
+    width: 100%;
+  }
+
+  #form-desc {
```

Comment:
- [ ] Like the previous reviewer mentioned kindly make sure your `footer` looks as close as possible to the Figma design especially the text. 🙏 

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/66992901/193105885-1671392a-d4d1-4031-a302-d3cbaa87bb01.PNG)  |  ![image](https://user-images.githubusercontent.com/74814780/193279621-c321fadf-76a7-4263-b8f5-dd9fb6f13c24.png)

Reply:
![Capture](https://user-images.githubusercontent.com/109624418/193397961-f255c324-0ee3-4fde-a8ba-59a28d28ce18.JPG)

Corrected and updated.